app/models/execution_environment.rb

```diff
@@ -70,21 +76,48 @@ def valid_test_setup?
           attribute: I18n.t('activerecord.attributes.execution_environment.testing_framework')))
     end
   end
-  private :valid_test_setup?
 
   def validate_docker_image?
     docker_image.present? && !Rails.env.test?
   end
-  private :validate_docker_image?
 
   def working_docker_image?
-    runner = Runner.for(author, self)
-    output = runner.execute_command(VALIDATION_COMMAND)
-    errors.add(:docker_image, "error: #{output[:stderr]}") if output[:stderr].present?
-  rescue Runner::Error::NotAvailable => e
-    Rails.logger.info("The Docker image could not be verified: #{e}")
+    sync_runner_environment
+    retries = 0
+    begin
+      runner = Runner.for(author, self)
```

Comment:
Here we request a new runner id via the `before_validation` hook of the Runner. This fails if we want to create an execution environment with an empty prewarming pool size (as db:seed does).
Should we adjust the prewarming pool size on the CodeOcean side for the moment of testing or provide an extra route or flag for test behaviour on the Poseidon side?

Reply:
Mh, that's a good finding I forgot (I actually tested the way to increase the pool size from 0 to another number but not the other way round). What if we skip the validation in case the new pool size is 0? The image cannot be used when the pool is empty and the next edit (which will increase the pool size) will allow verification.

Reply:
That sounds great!

Reply:
Okay, great. I did so with b8e8f50 and additionally tested seeding manually. As a result, I also created #1082